docs/source/user/data/data_formats_help.rst

```diff
@@ -0,0 +1,217 @@
+can be removed *except the last line*, and only the first three columns
+(*Qx, Qy,* and *I(Qx,Qy)*) are actually required.
+
+Data values have the same meanings and units as for `1D SAS Formats`_.
+
+*2D image data* can be translated into 2D 'pseudo-data' using the
+:ref:`Image_Viewer_Tool`, but this should only be done with an abundance of
+caution.
+
+**Examples of these formats can be found in the \\example_data\\2d_data sub-folder
+in the SasData distribution.**
+
+.. ZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZ
+
+1D SESANS Format
+----------------
+
+SasData will read ASCII ('text') files in a prototype SESANS
+standard format with the extensions .SES or .SESANS (case-insensitive).
+
+The file format starts with a list of name-value pairs which detail the general
+experimental parameters necessary for fitting and analyzing the data. This list
+should contain all the information necessary for the file to be 'portable'
+between users.
+
+Following the header are up to 8 space-delimited columns of experimental
+variables of which the first 4 columns are required. In order, these are:
+
+- Spin-echo length (z, in Angstroms)
+- Depolarization (:math:`log(P/P_0)/(lambda^2 * thickness)`, in Angstrom :sup:`-1` cm :sup:`-1`\ )
+- Depolarization error (also in in Angstrom :sup:`-1` cm :sup:`-1`\ ) (i.e. the measurement error)
+- Spin-echo length error (:math:`\Delta`\ z, in Angstroms) (i.e. the experimental resolution)
+- Neutron wavelength (:math:`\lambda`, in Angstroms)
+- Neutron wavelength error (:math:`\Delta \lambda`, in Angstroms)
+- Normalized polarization (:math:`P/P_0`, unitless)
+- Normalized polarization error (:math:`\Delta(P/P_0)`, unitless) (i.e. the measurement error)
+
+**Examples of this format can be found in the \\example_data\\sesans_data sub-folder
+in the SasData distribution.**
+
+.. ZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZ
+
+Coordinate Formats
+------------------
+
+The :ref:`SANS_Calculator_Tool` in SasView recognises ASCII ('text') files
+containing coordinate data (a grid of 'voxels') with the following extensions
+(which are not case-sensitive):
+
+*  .PDB (`Protein Data Bank format <https://www.wwpdb.org/documentation/file-format>`_)
+*  .OMF (`OOMMF micromagnetic simulation format <https://math.nist.gov/oommf/doc/userguide20a2/userguide/Vector_Field_File_Format_OV.html>`_)
+*  .SLD (Spin-Lattice Dynamics simulation format)
+
+In essence, coordinate formats specify a location and one or more properties of
+that location (e.g. what it represents, its volume, or magnetisation, etc). The
+PDB/OMF/SLD formats all use a rectangular grid of voxels.
+
+The .STL coordinate format is not currently supported by SasView.
+
+**Examples of these formats can be found in the \\example_data\\coordinate_data
+sub-folder in the SasData distribution.**
+
```

Comment:
I may be wrong but isn't VTK format missing for GSC?

Reply:
It's questionable if this should even be in the sasdata documentation while the coordinate data readers and data objects are still in sasview (see https://github.com/SasView/sasview/issues/2656).

Reply:
Good point but to @wpotrzebowski point, I thought GSC is in fact where VTK is being used by Dirk for his magnetic stuff (but only a  specific version?)